tests/reporters/test_custom_x_reporter.py

```diff
@@ -28,14 +28,16 @@
 
 
 account_fake = accounts.add("023861e2ceee1ea600e43cbd203e9e01ea2ed059ee3326155453a1ed3b1113a9")
-with mock.patch("builtins.input", side_effect=["", ""]):
-    try:
-        account = ChainedAccount("fake").add(
-            "fake", 80001, "023861e2ceee1ea600e43cbd203e9e01ea2ed059ee3326155453a1ed3b1113a9"
-        )
-    except Exception as e:
-        if e.args[0] == "Account fake already exists ":
-            account = ChainedAccount("fake")
+try:
```

Comment:
Same as other comment. See if your desired fake account already exists first with `find_account`, then create it if needed